L16-io/homework/src/main/java/ru/otus/dataprocessor/ResourcesFileLoader.java

```diff
@@ -0,0 +1,38 @@
+package ru.otus.dataprocessor;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import ru.otus.model.Measurement;
+
+public class ResourcesFileLoader implements Loader {
+
+    private static final Logger log = LoggerFactory.getLogger(ResourcesFileLoader.class);
+    private final Path fileName;
+
+    public ResourcesFileLoader(String fileName) {
+        this.fileName = Paths.get("src/test/resources/%s".formatted(fileName));
+    }
+
+    @Override
+    public List<Measurement> load() {
+        List<Measurement> measurements;
+
+        File jsonFile = fileName.toFile();
+        final var mapper = ObjectMapperFactory.mapper;
```

Comment:
Какая-то странная фабрика получилась.
Вряд ли создание здесь `ObjectMapper` скажется на производительности, чтобы везде переиспользовать один `ObjectMapper`:
```suggestion
        final var mapper = new ObjectMapper();
```

К тому же здесь можно сделать так:
```java
ObjectMapperFactory.mapper = new ObjectMapper();
```
и сильно что-то сломать.